src/client/cli/util.js

```diff
@@ -1,8 +1,13 @@
 // @flow
 
+const { clone, set } = require('lodash')
+const fs = require('fs')
 const Multihash = require('multihashes')
 const { JQ_PATH } = require('../../metadata/jqStream')
 const childProcess = require('child_process')
+const sshTunnel = require('tunnel-ssh')
+const yaml = require('js-yaml')
```

Comment:
I'd prefer to avoid config format proliferation this early in the process, even if yaml and JSON can technically the equivalent